### PR TITLE
Add 'pull_reqest' workflow and 'test' steps

### PR DESCRIPTION
**Pull request by aeifn**

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ on:
     tags:
       - 'v*'
   workflow_dispatch:
+  pull_request:
```
Comment (kabalin) [inline .github/workflows/docker-image.yml:10]: This was run twice in fact. Let's divide into two jobs: 
* build: runs CI only (ends after current Build step) 
* app-push: runs the rest of the steps in separate job only on on requires branches changes (master and en)

**Review by kabalin — CHANGES_REQUESTED** [review]

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.10.0
+
+      - name: Build
```
Comment (kabalin) [inline .github/workflows/pull-request.yml:23]:
```suggestion
      - name: CI
```

File: .github/workflows/pull-request.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+name: Pull request check
```
Comment (kabalin) [inline .github/workflows/pull-request.yml:1]:
```suggestion
name: Pull request CI test
```

**Review by kabalin — APPROVED** [review]

**Comment by kabalin** [comment]
Please consider job naming suggestions and squash commits before on merging.

File: .github/workflows/pull-request.yml (hunk 2 above)
Comment (aeifn) [reply .github/workflows/pull-request.yml:23]: This step actually runs only `npm run build`, so it's best to leave it as is.

**Pull request merged**